test/end-to-end/company.js

```diff
@@ -39,6 +39,8 @@ describe("Company endpoint", () => {
         description: "For Testing Purposes",
         contacts: ["geral@niaefeup.pt", "229417766"],
         jobType: "SUMMER INTERNSHIP",
+        jobMinDuration: 1,
+        jobMaxDuration: 6,
```

Comment:
This applies to the rest of the lines that are like these: when the tests are refactored (see #169 ) these should be put in the offer creation utility.